hazelcast-jet-core/src/main/java/com/hazelcast/jet/impl/JetService.java

```diff
@@ -51,21 +52,25 @@
 import java.util.concurrent.atomic.AtomicInteger;
 import java.util.function.Supplier;
 
+import static com.hazelcast.spi.properties.GroupProperty.SHUTDOWNHOOK_POLICY;
 import static java.util.concurrent.TimeUnit.SECONDS;
 
 public class JetService
         implements ManagedService, ConfigurableService<JetConfig>, PacketHandler, MembershipAwareService,
         LiveOperationsTracker {
 
+    public static final String SERVICE_NAME = "hz:impl:jetService";
     public static final int MAX_PARALLEL_ASYNC_OPS = 1000;
+    public static final HazelcastProperty JET_SHUTDOWN_HOOK_ENABLED =
```

Comment:
can we move this `GroupProperty` ?

Reply:
moved it. I was not aware we have jet specific group property class